pkg/network/ebpf/c/protocols/tls/java/erpc_handlers.h

```diff
@@ -127,55 +127,55 @@ int kprobe_handle_connection_by_peer(struct pt_regs *ctx) {
 */
 SEC("kprobe/handle_async_payload")
 int kprobe_handle_async_payload(struct pt_regs *ctx) {
```

Comment:
Why keeping this function if everything is commented out?

Reply:
java tls is not enabled, but removing the function means that we'll need to do further modifications to the code
since this is a custom version I see it redundant and less relevant at the moment